lib/msal-common/src/request/RequestParameterBuilder.ts

```diff
@@ -383,6 +384,30 @@ export class RequestParameterBuilder {
         this.parameters.set(AADServerParamKeys.X_MS_LIB_CAPABILITY, ThrottlingConstants.X_MS_LIB_CAPABILITY_VALUE);
     }
 
+    /**
+     * Add stk_jwk thumbprint to query params
+     * @param stkJwkKid
+     */
+    addStkJwkThumbprint(stkJwkKid: string): void {
```

Comment:
What happens if this is null/empty? Do we handle that where we call this?

Reply:
It just doesn't get added, it most likely won't get called if the stk_jwk hasn't been set, we do check before we call this.

Reply:
Thanks